apps/tlv-app/grails-app/assets/javascripts/menus/annotations-map.js

```diff
@@ -95,257 +103,274 @@ function applyAnnotationStyle() {
-	radiusInput.prev().html("Radius");
-
-	var stroke = style.getStroke();
-	var strokeColor = ol.color.asArray(stroke.getColor());
-  	var strokeWidth = stroke.getWidth();
-
-	switch (feature.getGeometry().getType()) {
-		case "Circle":
-			radiusInput.prev().html("Radius (meters)");
-			radius = getCircleRadius(feature.getGeometry());
+	var radiusInput = $( '#radiusInput' );
+	radiusInput.prop( 'disabled', false );
+	radiusInput.prev().html( 'Radius' );
+	
+	var stroke      = style.getStroke();
+	var strokeColor = ol.color.asArray( stroke.getColor() );
+	var strokeWidth = stroke.getWidth();
+	
+	switch( feature.getGeometry().getType() ) {
+		case 'Circle':
+			radiusInput.prev().html( 'Radius (meters)' );
+			radius = getCircleRadius( feature.getGeometry() );
 			break;
-		case "Point":
+		case 'Point':
 			var image = style.getImage();
-			fillColor = ol.color.asArray(image.getFill().getColor());
-			radius = image.getRadius();
-			radiusInput.prev().html("Radius (pixels)");
-			var stroke = style.getImage().getStroke();
-			strokeColor = ol.color.asArray(stroke.getColor());
+			fillColor = ol.color.asArray( image.getFill().getColor() );
+			radius    = image.getRadius();
+			radiusInput.prev().html( 'Radius (pixels)' );
+			var stroke  = style.getImage().getStroke();
+			strokeColor = ol.color.asArray( stroke.getColor() );
 			strokeWidth = stroke.getWidth();
 			break;
 		default:
-			radiusInput.prop("disabled", true);
+			radiusInput.prop( 'disabled', true );
 			break;
 	}
-
-	var fillColorHex = rgbToHex(fillColor[0], fillColor[1], fillColor[2]);
-	$("#fillColorInput").val(fillColorHex);
-	$("#fillTransparencyInput").val(fillColor[3]);
-
-	radiusInput.val(radius || 0);
-
-	var strokeColorHex = rgbToHex(strokeColor[0], strokeColor[1], strokeColor[2]);
-	$("#strokeColorInput").val(strokeColorHex);
-	$("#strokeTransparencyInput").val(strokeColor[3]);
-	$("#strokeWidthInput").val(strokeWidth);
-
+	
+	var fillColorHex = rgbToHex( fillColor[ 0 ], fillColor[ 1 ], fillColor[ 2 ] );
+	$( '#fillColorInput' ).val( fillColorHex );
+	$( '#fillTransparencyInput' ).val( fillColor[ 3 ] );
+	
+	radiusInput.val( radius || 0 );
+	
+	var strokeColorHex = rgbToHex( strokeColor[ 0 ], strokeColor[ 1 ], strokeColor[ 2 ] );
+	$( '#strokeColorInput' ).val( strokeColorHex );
+	$( '#strokeTransparencyInput' ).val( strokeColor[ 3 ] );
+	$( '#strokeWidthInput' ).val( strokeWidth );
+	
 	var properties = feature.getProperties();
-	$( "#confidenceSelect option[value=" + properties.confidence + "]" ).prop( "selected", true );
-	$( "#typeInput" ).val( properties.type );
-	$( "#usernameInput" ).val( properties.username );
+	$( '#confidenceSelect option[value=' + properties.confidence + ']' ).prop( 'selected', true );
+	$( '#typeInput' ).val( properties.type );
+	$( '#usernameInput' ).val( properties.username );
 }
 
-var pageLoadAnnotation = pageLoad
-pageLoad = function() {
+var pageLoadAnnotation = pageLoad;
+pageLoad               = function() {
```

Comment:
Too many spaces? Probably something I did.

Reply:
Nah, that's my editor. Keeps all the `=` inline